StarterProject/pageobjects/common.js

```diff
@@ -0,0 +1,87 @@
+import Page from "./page.js";
+
+export default class Common extends Page {
+
+    constructor() {
+        super();    
+    }
+
+    closeWelcomeModal(){
+        let close = "div.emailPopupRefresh img";
+
+        if (this.isWelcomeModalPresent()){
+            browser.element(close).click;
+        }
+    }
+
+    closeWelcomeModalWhenPresent(){
+        let emailModal ="#monetate_allinone_lightbox";
+        browser.call(()=>{
+            browser.isVisible(emailModal).then(()=>{
+                this.closeWelcomeModal();
+            });
+        });
+    }
+
+    isWelcomeModalPresent(){
+
+        let emailModal ="#monetate_allinone_lightbox";
```

Comment:
`let` is valid but unless you're going to change the value `const` would be safer